tests/data_multiclassification/test.txt

```diff
@@ -0,0 +1,30 @@
+Die Dame trägt einen riesigen Klotz .	riesigen Klotz	Dimension
+Opa hat ein pinkes Auto .	pinkes Auto	Farbe
+Wir fahren ein eckiges Gefährt .	eckiges Gefährt	Form
+Der Mann hat ein breites Kreuz.	breites Kreuz	Dimension
+Oma schaut eine rote Blume an .	rote Blume	Farbe
+Es gibt dort eine eckige Zeitschrift .	eckige Zeitschrift	Form
+Das ist eine riesige Schlange .	riesige Schlange	Dimension
+Er trägt einen spitzen Hut .	spitzen Hut	Form
+Der Mann hat ein schmales Kinn .	schmales Kinn	Dimension
+Ihr rundes Gesicht ist schön .	rundes Gesicht	Form
+Er hat eine pinke Jacke .	pinke Jacke	Farbe 
+Das ist eine kurze Schlange .	kurze Schlange	Dimension
+Der Postbote bringt ein rechteckiges Paket .	rechteckiges Paket	Form
+Die Dame trägt einen kurzen Rock.	kurzen Rock	Dimension
+Die Dame trägt einen langen Rock .	langen Rock	Dimension
+Das ist ein riesiges Haus.	riesiges Haus	Dimension
+Lena kauft ein braunes Kleid .	braunes Kleid	Farbe
+Er hat eine hellblaue Jacke .	hellblaue Jacke	Farbe 
+Lena kauf einen riesigen Bären .	riesigen Bären	Dimension
+Wir laufen auf einer runden Bahn .	runden Bahn	Form
+Lena kauft ein schwarzes Kleid .	schwarzes Kleid	Farbe
+Die Dame trägt einen langes Kleid.	langes Kleid	Dimension
+Der Mann hat ein breites Gesäß .	breites Gesäß	Dimension
+Er hat eine gelbe Jacke .	gelbe Jacke	Farbe 
+Warum kann ich keine rosane Farbe haben ?	rosane Farbe	Farbe
+Sie kauft einen gelben Rock .	gelben Rock	Farbe
+Du musst noch einen kleinen Augenblick warten .	kleinen Augenblick	Dimension
```

Comment:
I know that this is a toy data set but why is it Dimension? Similar to _kurzer Zeit_ in train.txt.

Reply:
I don't know, what else is kurze Zeit? does Dimension not fit at all? there are more examples like that
kurzer Abend
kurze Minute 
...
we can replace them if you think we should. maybe you have some examples that work. After some time I was a bit fed up of constructing example sentences so I did not think it trough very deeply.

Reply:
maybe _Zeit_ would be better. So that we distinguish between _schmales Haus_ and _kurzer Augenblick_. 
But since it is only a small toy data set we can leave it as it is, I guess

Reply:
OK yes, makes sense. Let's see whether we can use it for some tests. If the data is not easily separable we can try to change it into easier data.